mgnify_pipelines_toolkit/analysis/rawreads/study_summary_generator.py

```diff
@@ -162,16 +167,20 @@ def parse_one_func_file(
     ).set_index("function")
     res_df = res_df.fillna(0)
 
-    validate_dataframe(res_df, FunctionProfileSchema, str(func_file))
+    if res_df.shape[0] > 0:
```

Comment:
I don't think it's necessary because it's subjective but martin has influenced me to instead do something like this
```suggestion
    if res_df.shape[0] < 0:
        continue
```
that way you don't need to indent the rest of the code block for being in the conditional, which helps with readability! Not at all important though

Reply:
Thanks for the tip